Automattic/newspack-block-theme
# feat: add footer patterns and update template part

### All Submissions:






### Changes proposed in this Pull Request:

This PR adds 3 footer patterns and has the Footer template part to use style 1 by default. This will allow with WP 6.6 the ability to easily switch between footer styles.

https://github.com/Automattic/newspack-block-theme/assets/177929/7e826025-fd51-47f0-8ba9-2979934fab22

__Note: I've also removed ALL the useless patterns coming from Core, WC, Jetpack, and other plugins. And I'm using a whitelist instead (so far, all I can think of is newspack and publisher media kit)__

### How to test the changes in this Pull Request:

1. Install WP 6.6 (I don't think the easy switch exists in 6.5)
2. Switch to this branch
3. Make sure the footer is the right one by default.
4. Try to switch between styles and save

### Other information:
